CMakeLists.txt

```diff
@@ -1,10 +1,22 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(kuka_lwr4_description)
-find_package(catkin REQUIRED)
+
+find_package(catkin REQUIRED COMPONENTS xacro)
 
 catkin_package(CATKIN_DEPENDS xacro)
 
-foreach(dir calibration defs meshes robots)
-   install(DIRECTORY ${dir}/
-      DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}/${dir})
+# generate and install URDFs from xacro files
+# source: http://wiki.ros.org/xacro#Building_from_CMakeLists.txt
+
+set(xacro_files
+    ${CMAKE_CURRENT_SOURCE_DIR}/robots/lwr_ft.urdf.xacro
+    ${CMAKE_CURRENT_SOURCE_DIR}/robots/lwr_pen.urdf.xacro
+    ${CMAKE_CURRENT_SOURCE_DIR}/robots/lwr_sdh.urdf.xacro
+)
+
+xacro_add_files(${xacro_files} REMAP collision_simple:=0 TARGET media_files INSTALL DESTINATION robots)
```

Comment:
@christian-rauch Add INORDER here.

Reply:
I did not consider this at first because the documentation says:
`Because --inorder processing is much more powerful, in future versions beyond Jade, the new processing style will become default [...]`
However, this seems to only be applied to `xacro_add_xacro_file` and not `xacro_add_files`.